standalone-metastore/metastore-server/pom.xml

```diff
@@ -737,7 +737,7 @@
         <configuration>
           <api>JDO</api>
           <verbose>false</verbose>
-          <log4jConfiguration>${basedir}/src/main/resources/datanucleus-log4j.properties
+          <log4jConfiguration>${basedir}/src/main/resources/datanucleus-log4j2.xml
```

Comment:
Just wonder why XML is preferred here

Reply:
Based on the datanucleus documentation, log4j2 was introduced as part of 6.0 verison https://www.datanucleus.org/products/accessplatform_6_0/logging.html and there are different format for log4j (properties file) vs log4j2 (xml file) provided by them.

older datanucleus-log4j.properties was exact same as mentioned in documentation other than:
log4j.appender.A1.File=datanucleus.log => log4j.appender.A1.File=**target/datanucleus.log**